src/main/java/com/palantir/gradle/graal/SharedLibraryTask.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.gradle.graal;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.gradle.api.tasks.TaskAction;
+
+/**
+ * Runs GraalVM's native-image for shared library command with configured options and parameters.
+ */
+public class SharedLibraryTask extends BaseGraalCompileTask {
+
+    public SharedLibraryTask() {
+        setDescription(
+                "Runs GraalVM's native-image for shared library command with configured options and parameters."
```

Comment:
this sentence doesn't quite parse for me

Reply:
Would you help me with a more consistent sentence here. My english is not that good

Reply:
of course:
```
Runs GraalVM's native-image command configured to produce a shared library.
```
(same should go to the class javadoc)